app/src/main/res/values/strings.xml

```diff
@@ -165,11 +165,21 @@
     <string name="no_accounts_found_desc"><![CDATA[ It looks like "<b><font color="#39CBFC"> %1$s </font> </b>" has not been listed on Stax yet or its an invalid entry.<br/>Try searching for something else.]]></string>
     <string name="cant_find_institution">Can\'t find your financial institution?</string>
     <string name="inform_us_here">Inform us here</string>
+
+    <string name="enter_institution_name">Enter Bank, service or account name</string>
+    <string name="enter_shortcode">Enter shortcode</string>
+    <string name="inform_stax_desc">Inform Stax on your financial institution</string>
```

Comment:
This is too long for a dialog title. On smaller devices, it will span more than one line.